### PR TITLE
 Convert AJM jitsiMeet factory to a provider so it can be configured i…

### PR DESCRIPTION
**Pull request by wabeeler**

### CODE REVIEW

File: src/index.coffee [hunk 1]
```diff
@@ -18,26 +18,32 @@ for modName, mod of APP
...
-      #AngularServiceGenerator.wireUpEvents EventAdapter, ajmModule, APP, eventMappings
+  vm.config =
+    wireEvents: true
+
+  vm.wireEvents = (shouldWireEvents) ->
+    vm.config.wireEvents = shouldWireEvents
+
+  vm.$get = (EventAdapter) ->
+    APP.wireUpEvents = ->
+      if !eventsWired && vm.config.wireEvents
```
Comment (dwilson6) [inline src/index.coffee]: What about moving the check of vm.config.wireEvents to line 44 where wireUpEvents is invoked?  That way wireUpEvents is more like autoWireUpEvents and you can choose if you want to wire up the events later.

File: src/index.coffee [hunk 2]
```diff
@@ -18,26 +18,32 @@ for modName, mod of APP
...
+        console.log "--- WIRING Jitsi EVENTS"
+        #AngularServiceGenerator.wireUpEvents EventAdapter, ajmModule, APP, eventMappings
```
Comment (dwilson6) [inline src/index.coffee]: You can get rid of this

**Comment by rdsmallwood928** [comment]
Looks good!

File: src/index.coffee (hunk 1 above)
Comment (wabeeler) [inline src/index.coffee]: Good catch

**Pull request merged**